solr/core/src/java/org/apache/solr/util/SolrCLI.java

```diff
@@ -828,7 +665,54 @@ public static <T> T pathAs(Class<T> clazz, String jsonPath, Map<String, Object>
   }
 
   /**
-   * Helper function for reading an Object of unknown type from a JSON Object tree.
+   * Helper function for reading an Object of unknown type from a JSON Object tree stored in
+   * NamedList format.
+   *
+   * <p>To find a path to a child that starts with a slash (e.g. queryHandler named /query) you must
```

Comment:
The term queryHandler is an ancient Solr word; requestHandler is the general replacement.